### PR TITLE
Bug Correct - After #98

### LINKED ISSUE #80: [TO_IMPLEMENT] Try catch on file delete after update image
Image

### LINKED ISSUE #86: Block and Unblock user information sent is too much
- We are sending all user information

### LINKED ISSUE #89: Nickname must be unique
- Whenever user updates nick either in UserValidation or in Settings there must be a validation to check if nickname is unique.

### LINKED ISSUE #92: [NICK CHANGE]
when there's the change nick modal, if the user decides to change nick it MUST update the token, because the token contains the old nick, ruining the sent message logic and the admin commands logic

### PR DESCRIPTION
**Pull request by ncameiri42**
-Duplicated Nick validation
-Disconnect old Socket when login other session
-User already in use alert
-Protect NULL avatar delete
-Block and unblock user with limited info

### CODE REVIEW

File: frontend/src/views/ProfileView.vue [hunk 1]
```diff
@@ -258,10 +259,15 @@ async function saveSettings() {
       });
       if (response.ok) {
         let data = await response.json();
-        console.log("NEW URL "+data.newAvatar);
+        console.log("NEW URL "+data.newAvatar);        
+      }
+      //Caso o Nick já esteja em Uso
+      else if(response){
+        userProfile.value.nick = oldNick;
```
Comment (gleal42) [inline frontend/src/views/ProfileView.vue:267]:
```suggestion
        userProfile.value.nick = oldNick;
        alert("User already in Use")
```

Comment (gleal42) [reply frontend/src/views/ProfileView.vue:267]: Just so it is the same as UserValidation

**Review by gleal42 — CHANGES_REQUESTED** [review]
Great job!

In order to close the #92 issue, since nick can also be changed in ProfileView the update token should also be there.

To close #89  Same for the alert when nick is duplicated.

**Comment by gleal42** [comment]
Apart from that. Everything looks great! Love the block Unblock feature on channels. Looks really clean.

**Comment by ncameiri42** [comment]
Update token on ProfileView applied and alert

**Review by J0Santos — APPROVED** [review]

**Review by gleal42 — APPROVED** [review]
LGTM

**Pull request merged**